packages/agoric-cli/src/lib/chain.js

```diff
@@ -96,16 +95,21 @@ harden(execSwingsetTransaction);
 /**
  *
  * @param {import('./rpc.js').MinimalNetworkConfig} net
- * @returns {Promise<import('@agoric/cosmic-proto/swingset/swingset.js').Params>}
  */
-export const fetchSwingsetParams = async net => {
-  const { rpcAddrs } = net;
-  const rpcEndpoint = rpcAddrs[0];
-  const client = await agoric.ClientFactory.createRPCQueryClient({
-    rpcEndpoint,
-  });
-  const { params } = await client.agoric.swingset.params();
-  return params;
+// TODO fetch by HTTP instead of shelling out https://github.com/Agoric/agoric-sdk/issues/9200
+export const fetchSwingsetParams = net => {
+  const { chainName, rpcAddrs } = net;
+  const cmd = [
+    `--node=${rpcAddrs[0]}`,
+    `--chain-id=${chainName}`,
+    'query',
+    'swingset',
+    'params',
+    '--output',
+    '--json',
```

Comment:
`--json` is not a flag, it is an argument to `--output`, like `--output json` or `--output=json` or `-ojson` or `-o json`:
```suggestion
    'json',
```